README.md

```diff
@@ -43,7 +43,9 @@ separately on-prem or leveraged as managed services.
 Assuming kubectl context points to the correct kubernetes cluster, first create kubernetes secrets that contain MySQL and Neo4j passwords. 
 
 ```(shell)
-kubectl create secret generic mysql-secrets --from-literal=mysql-root-password=datahub
+kubectl create secret generic mysql-secrets --from-literal=mysql-root-password=datahub \
+											--from-literal=mysql-replication-password=datahub \
+											--from-literal=mysql-password=datahub
```

Comment:
@mkaramuk only `mysql-root-password` and `mysql-password` are required, also no line break needed here